jobManager.py

```diff
@@ -62,9 +67,16 @@ def __manage(self):
 
             if id:
                 job = self.jobQueue.get(id)
+
+                # job could no longer exist if it was completed by someone else
+                if job == None:
+                    continue
+
                 if not job.accessKey and Config.REUSE_VMS:
                     id, vm = self.jobQueue.getNextPendingJobReuse(id)
                     job = self.jobQueue.get(id)
+                    if job == None:
```

Comment:
this fix does seem to address the issue mentioned here https://github.com/autolab/Tango/issues/138.